classes/class.pmproemail.php

```diff
@@ -511,8 +511,6 @@ function sendRefundedEmail( $user = NULL, $invoice = NULL ) {
 				'accountnumber' => hideCardNumber($invoice->accountnumber),
 				'expirationmonth' => $invoice->expirationmonth,
 				'expirationyear' => $invoice->expirationyear,
```

Comment:
These variables may be used in custom templates.
```suggestion
				'expirationyear' => $invoice->expirationyear,
 				 login_link' => pmpro_login_url(),
 				'login_url' => pmpro_login_url(),
```